<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PortSetBox</class>
 <widget class="QDialog" name="PortSetBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Port Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QComboBox" name="stopBitsBox">
     <item>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">1.5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">2</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Flow Control</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Parity</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Stop Bits</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="flowControlBox">
     <item>
      <property name="text">
       <string notr="true">None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">RTS/CTS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">XON/XOFF</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="dataBitsBox">
     <item>
      <property name="text">
       <string notr="true">5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">8</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Data Bits</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="paritBox">
     <item>
      <property name="text">
       <string notr="true">None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Even</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Odd</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Space</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Mark</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="autoDTRBox">
     <property name="text">
      <string>Auto DTR</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PortSetBox</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PortSetBox</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
